dominion/dominion_ui/src/components/Hero/ProfileHero.js

```diff
@@ -0,0 +1,129 @@
+    left: 'unset !important',
+    top: 'unset !important',
+    [theme.breakpoints.up('md')]: {
+      position: 'absolute !important',
+      right: '50px',
+      width: '50% !important',
+      height: '460px !important',
+      maxHeight: '460px !important',
+      maxWidth: '829px !important'
+    },
+    [theme.breakpoints.up('lg')]: {
+      right: '9.375rem'
+    }
+  },
+  caption: {
+    color: '#8d8d8c',
+    fontSize: '0.75em',
+    textTransform: 'capitalize',
+    paddingTop: theme.spacing.unit,
+    paddingBottom: theme.spacing.unit
+  },
+  captionItem: {
+    display: 'inline-block'
+  },
+  release: {
+    display: 'none',
+    [theme.breakpoints.up('md')]: {
+      color: '#8d8d8c',
+      fontSize: '0.95em',
+      position: 'absolute',
+      bottom: '19%',
+      display: 'inline-block',
+      right: '4%'
+    },
+    [theme.breakpoints.up('lg')]: {
+      bottom: '15%',
+      right: '6.8%'
+    }
+  }
+});
+class ProfileHero extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { level: '', geoid: '' };
+  }
+
+  async componentDidMount() {
+    // get level from mapit
+    const geoid = window.geography.full_geoid;
+    const api = createAPI();
+    const level = await api.getGeoLevel(geoid);
+    this.setState({ level, geoid });
+  }
+
+  render() {
+    const { population } = window.population;
+    const { populationDensity } = window.population_density;
+    const { level, geoid } = this.state;
+    const { classes } = this.props;
+
+    return (
+      <Hero>
+        <HeroTitleGrid quater>
+          <HeroTitle breakWord small>
+            {window.geography.short_name}
+          </HeroTitle>
+          <Typography variant="body2" className={classes.caption} component="p">
+            {level}{' '}
+            {window.captionItems.length ? (
+              <Typography variant="body" className={classes.captionItem}>
+                in{' '}
+                {window.captionItems.slice(0, -1).map(item => (
+                  <span>
+                    <a href={`/profiles/${item.geoid}`}>{item.name}</a>{' '}
+                  </span>
+                ))}
+              </Typography>
+            ) : null}
+          </Typography>
+          <HeroDetail label="Population">{population}</HeroDetail>
+          <HeroDetail small label="square kilometers">
+            {window.geography.square_kms}
+          </HeroDetail>
+          <HeroDetail small label="people per square kilometer">
+            {populationDensity}
+          </HeroDetail>
+          <Search
+            handleIconClick={null}
+            isComparisonSearch
+            placeholder="Compare this with"
+            thisGeoId={geoid}
+            icon={searchIcon}
+          />
+        </HeroTitleGrid>
+        <Grid id="slippy-map" className={classes.map} />
+        <Typography variant="body2" className={classes.release}>
+          {window.primary_releases} {window.primary_releases_year}
```

Comment:
there is a `caption` that is concat `Census 2016`. I use it in releases section.

Reply:
Okay